[2025-04-04T09:08:12.013+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_images scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2025-04-04T09:08:12.055+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_images scheduled__2024-01-02T00:00:00+00:00 [queued]>
[2025-04-04T09:08:12.057+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T09:08:12.057+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-04-04T09:08:12.058+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T09:08:12.092+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): download_images> on 2024-01-02 00:00:00+00:00
[2025-04-04T09:08:12.108+0000] {standard_task_runner.py:55} INFO - Started process 2944 to run task
[2025-04-04T09:08:12.123+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'download_and_store_images', 'download_images', 'scheduled__2024-01-02T00:00:00+00:00', '--job-id', '732', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_collect.py', '--cfg-path', '/tmp/tmp9335uh97']
[2025-04-04T09:08:12.124+0000] {standard_task_runner.py:83} INFO - Job 732: Subtask download_images
[2025-04-04T09:08:12.151+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-04-04T09:08:12.217+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2025-04-04T09:08:12.312+0000] {task_command.py:388} INFO - Running <TaskInstance: download_and_store_images.download_images scheduled__2024-01-02T00:00:00+00:00 [running]> on host 91a2461e5846
[2025-04-04T09:08:12.622+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_and_store_images
AIRFLOW_CTX_TASK_ID=download_images
AIRFLOW_CTX_EXECUTION_DATE=2024-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-02T00:00:00+00:00
[2025-04-04T09:08:12.928+0000] {python.py:177} INFO - Done. Returned value was: [None, None]
[2025-04-04T09:08:13.015+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=download_and_store_images, task_id=download_images, execution_date=20240102T000000, start_date=20250404T090812, end_date=20250404T090813
[2025-04-04T09:08:13.085+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-04-04T09:08:13.201+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
